### DISCUSSION TITLE
[bug]: taping anything not working in the upper menu bar

### DISCUSSION THREAD

**Discussion by enriqueulloap** [community/community#64357] [Mobile]
### Select Topic Area

Bug

### Body

It's impossible to me use the upper menu bar, for instance to share something or to tap in the 3 points option. It is like a transparent overlay exist above this region and my taps are not perceived. This doesn't happen with other apps with this upper menu bar like whatsapp. I'm open to share any details you need to solve this, I would like to have a fully functional GitHub app

**Comment by DSDmark** [comment]
Hi @enriqueulloap,

Can you **specific more** about the details of what you want? For better understanding 😅. Also, you can add some **screenshots** or **error** which you were facing 👍, like:

- which OS and phone model version you are using.
- in which section of area you are facing that issue in app.
- how to reproducer the error or issue. Add detail about this.
- make your discussion **title** something like this `[bug]: taping anything not working in the upper menu bar`. It will catch the attention of developer team.
- And if you have more detail, then it's much better.

**Comment by enriqueulloap** [comment]
Hi @DSDmark, sure:

**which OS and phone model version you are using:** Android 10 QKQ1.191215.002 + Redmi Note 9 Pro

**in which section of area you are facing that issue in app:** here, in the upper menu bar where the share button lives:
![WhatsApp Image 2023-08-21 at 09 33 08](https://github.com/community/community/assets/1735422/f5dba304-ce12-4aa0-b803-5ba7ac2e974c)

**how to reproducer the error or issue:** it's just to open any element related to development (merges, approvals, etc.) in the Notifications screen. In the next screen, tapping on the buttons "back", "share" and "3 points" do nothing.

**And if you have more detail, then it's much better:** other parts of the application work just fine. For instance, in this screen, the share button works just fine:
![WhatsApp Image 2023-08-21 at 09 37 54](https://github.com/community/community/assets/1735422/bdd70a85-2692-4e92-879b-bd9d0ac90700)

**Reply by enriqueulloap** [reply]
I've solved the issue. It was caused by having enabled the option "Hide notch" in the Display settings.

**Comment by 26e43ee** [comment]
Je veux que vous me connecter mon compte mobile money 652174057 a mon compte github

**Reply by Agus110500** [reply]
J

**Reply by 26e43ee** [reply]
Ok

**Comment by 26e43ee** [comment]
Ok vous pouvez me joindre via WhatsApp ou 694841595
